RELEASE.md

```diff
@@ -2,6 +2,8 @@
 
 The Kubernetes Template Project is released on an as-needed basis. The process is as follows:
 
+1. Update `pkg/consts/consts.go` with the new semver tag, for RC release we should use something like `v0.5.0-rc1`, for MAJOR or MINOR release we should use something like `v0.5.0`
+1. Run the following command `make generate` which will update generated docs with the correct version info. (Note that you can't test with these YAMLs yet as they contain references to elements which wont exist until the tag is cut and image is promoted to production registry.)
```

Comment:
Recommend leaving the testing note out as you can at least install the CRDs without waiting for images to get published
```suggestion
1. Run the following command `make generate` which will update generated docs with the correct version info. 
```